site/en/blog/lighthouse-8-4/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: 'layouts/blog-post.njk'
+title: What's new in Lighthouse 8.4
+authors:
+  - brendankenny
+date: 2021-09-09
+description: >
+  Lighthouse 8.4 now guards against lazily-loading LCP images and failing FID without a mobile viewport.
+tags:
+  - new-in-lighthouse
+  - lighthouse
+  # - chrome-95
+---
```

Comment:
I think an introductory blurb to Lighthouse would be good. can c/p this: https://web.dev/lighthouse-whats-new-6.0/#:~:text=automated%20website%20auditing%20tool%20that%20helps%20developers

Reply:
Also can c/p the `To try the Lighthouse Node CLI, use the following commands: ...` bit